### PR TITLE
Introduce JobQuantize

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce minimum JobQuantize implementation for running
one-quantize command.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
From draft #70

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: src/Project/JobQuantize.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import {Job} from './Job';
+import {JobBase} from './JobBase';
+import {ToolArgs} from './ToolArgs';
+
+export class JobQuantize extends JobBase {
+  // TODO add quantize options
+  constructor() {
+    super();
+    this.jobType = Job.Type.tOptimize;
```
Comment (llFreetimell) [inline src/Project/JobQuantize.ts:25]:
```suggestion
    this.jobType = Job.Type.tQuantize;
```

**Review by llFreetimell — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**